addons/medical_statemachine/Statemachine.hpp

```diff
@@ -46,7 +46,7 @@ class ACE_Medical_StateMachine {
         onStateEntered = QFUNC(enteredStateUnconscious);
         class DeathAI {
             targetState = "Dead";
-            condition = QUOTE(!GVAR(AIUnconsciousness) && {!isPlayer _this});
+            condition = QUOTE(!(GVAR(AIUnconsciousness) || (_this getVariable [ARR_2(QQEGVAR(zeus,AIUnconsciousException), false)])) && {!isPlayer _this});
```

Comment:
```suggestion
            condition = QUOTE(!(_this getVariable [ARR_2(QQGVAR(AIUnconsciousness), GVAR(AIUnconsciousness))]) && {!isPlayer _this});
```

This would allow the variable to override the setting in both directions.

Reply:
This suggestion in this form is not working in-game. The part with ARR_2 is wrong, it is not using the correct variable that is set by the Zeus module. I do not have experience with comma operator, so for the time being, I' am unable to fix it.